google/logging/v2/logging_gapic.yaml

```diff
@@ -352,8 +352,11 @@ interfaces:
     bundling:
       thresholds:
         element_count_threshold: 100
-        request_byte_threshold: 1024 # 1 Kb
+        request_byte_threshold: 20480 # 20 Kb
```

Comment:
The maximum size of a single entries write request is 5MB. So we could bump this way up (and the corresponding flow control setting).
https://cloud.google.com/logging/quota-policy